README.rst

```diff
@@ -41,38 +45,39 @@ also symlinked to ``/opt/python``. Moreover, each installation of PyPy
 contains also a ``python`` symlink.
 
 All the following commands are equivalent and run the PyPy 2.7, version
-7.1.1. You can use whatever fits best in your build system:
+7.3.0. You can use whatever fits best in your build system:
 
-- ``/opt/pypy/pypy2.7-7.1.1/bin/pypy``
+- ``/opt/pypy/pypy2.7-7.3.0/bin/pypy``
 
-- ``/opt/pypy/pypy2.7-7.1.1/bin/python``
+- ``/opt/pypy/pypy2.7-7.3.0/bin/python``
 
-- ``/opt/python/pp271-pypy_41/bin/pypy``
+- ``/opt/python/pp273-pypy_73/bin/pypy``
 
-- ``/opt/python/pp271-pypy_41/bin/python``
+- ``/opt/python/pp273-pypy_73/bin/python``
 
-Similarly, these are the commands to run PyPy 3.6, version 7.1.1:
+Similarly, these are the commands to run PyPy 3.6, version 7.3.0:
 
-- ``/opt/pypy/pypy3.6-7.1.1/bin/pypy``
+- ``/opt/pypy/pypy3.6-7.3.0/bin/pypy``
 
-- ``/opt/pypy/pypy3.6-7.1.1/bin/python``
+- ``/opt/pypy/pypy3.6-7.3.0/bin/python``
 
-- ``/opt/python/pp371-pypy3_71/bin/pypy``
+- ``/opt/python/pp373-pypy36_pp73/bin/pypy``
 
-- ``/opt/python/pp371-pypy3_71/bin/python``
+- ``/opt/python/pp373-pypy36_pp73/bin/python``
 
 
 PEP 425 Compatibility tags
 ---------------------------
 
-``pp271-pypy_41`` and ``pp371-pypy3_71`` are the `PEP 425`_ compliant
+``pp273-pypy_73`` and ``pp373-pypy36_pp73`` are the `PEP 425`_ compliant
 compatibility tag. In particular:
 
 - ``pp`` stands for PyPy (as opposed to ``cp`` which is CPython)
 
-- ``271`` and ``371`` mean "Python [2|3]", "PyPy 7.1.x".
+- ``273`` and ``373`` mean "Python [2|3]", "PyPy 7.3.x".
 
-- ``pypy_41`` and ``pypy3_71`` are the binary ABI tags for the relevant
-  version of PyPy. You can probably ignore them.
+- ``pypy_73`` and ``pypy36_pp73`` (or before PyPy 7.3.0, ``pypy_41`` and
+  ``pypy3_71``) are the binary ABI tags for the relevant version of PyPy.
+  You can probably ignore them.
```

Comment:
I think it would be helpful to rephrase this in terms of "python tags" and "ABI tags". The python tag is controlled by pip/setuptools and is unfortunately `pp271`/`pp272`/`pp273`/`pp371`/`pp372`/`pp373`, it should be `pp27` or `pp36` (only the python version, not the pypy 71/72/73 version). The ABI tag is `pypy_41`/`pypy_73`/`pypy3_71`/`pypy3_72`/`pypy36_pp73`.

Reply:
Happy to change this, but you know a lot more about it and might be better at phrasing it. Feel free to push some changes to my branch/fork, before merging this PR.

Reply:
Probably best to let it go as-is, since anyhow the next version of pip should change the python tag, making a mess of whatever exists now. Luckily there are not that many pypy-specific wheels out there yet

Reply:
Does that mean it would be a good idea to withhold merging/releasing joerick/cibuildwheel#185 for now, until the pip/setuptools gets changed/fixed?

Reply:
I don't know. The source of the tag creation is https://github.com/pypa/packaging. The version at HEAD in that repo is not the same as version 19.0 which is currently vendored into pip https://github.com/pypa/pip/tree/master/src/pip/_vendor/packaging. I assume the idea is that packaging 20 (to be released) will be the one used in pip 20 (to be released) but I am not sure. I wanted to get 7.3.0 out (with the new ABI tags) before they release pip20, which I think will be some time in Jan 2020.

<details>
The 19.0 one in pip now shows these tags that start with `pp` (ignoring the cpython ones `cp*`:
```bash
$ ../pypy2-HEAD/bin/pypy -c"from packaging.tags import sys_tags;
       print('\n'.join([str(s) for s in sys_tags() if str(s).startswith('pp')]))"
pp273-pypy_73-manylinux2014_x86_64
pp273-pypy_73-manylinux2010_x86_64
pp273-pypy_73-manylinux1_x86_64
pp273-pypy_73-linux_x86_64
pp273-none-manylinux2014_x86_64
pp273-none-manylinux2010_x86_64
pp273-none-manylinux1_x86_64
pp273-none-linux_x86_64
pp273-none-any

$ ../pypy3.6-HEAD/bin/pypy3 -c"from packaging.tags import sys_tags;
        print('\n'.join([str(s) for s in sys_tags() if str(s).startswith('pp')]))"
pp373-pypy36_pp73-manylinux2014_x86_64
pp373-pypy36_pp73-manylinux2010_x86_64
pp373-pypy36_pp73-manylinux1_x86_64
pp373-pypy36_pp73-linux_x86_64
pp373-none-manylinux2014_x86_64
pp373-none-manylinux2010_x86_64
pp373-none-manylinux1_x86_64
pp373-none-linux_x86_64
pp373-none-any
```

The new HEAD version shows
```bash
$ ../pypy2-HEAD/bin/pypy -c"from packaging.tags import sys_tags;
           print('\n'.join([str(s) for s in sys_tags() if str(s).startswith('pp')]))"
pp27-pypy_73-manylinux2014_x86_64
pp27-pypy_73-manylinux2010_x86_64
pp27-pypy_73-manylinux1_x86_64
pp27-pypy_73-linux_x86_64
pp27-none-manylinux2014_x86_64
pp27-none-manylinux2010_x86_64
pp27-none-manylinux1_x86_64
pp27-none-linux_x86_64

$ ../pypy3.6-HEAD/bin/pypy3 -c"from packaging.tags import sys_tags; 
         print('\n'.join([str(s) for s in sys_tags() if str(s).startswith('pp')]))"
pp36-pypy36_pp73-manylinux2014_x86_64
pp36-pypy36_pp73-manylinux2010_x86_64
pp36-pypy36_pp73-manylinux1_x86_64
pp36-pypy36_pp73-linux_x86_64
pp36-none-manylinux2014_x86_64
pp36-none-manylinux2010_x86_64
pp36-none-manylinux1_x86_64
pp36-none-linux_x86_64

```

Reply:
OK, great; thank you for that! It's good to know, if tests soon start failing.

(Also makes me a bit happier that I chose the shortened `pp27_73-manylinux_pypy_x86_64` and `pp36_73-manylinux_pypy_x86_64` build identifiers for cibuildwheel :-) )